Docs/user_guide/getting_started.md

```diff
@@ -149,3 +149,51 @@ Browse the [User Guide](user_interface) section to find quick overview of the ap
 The application has a large and very friendly and helpful user community. We have poeple who will happy to help with simple questions, such as how to do a specific task in Slicer, and we have a large number of engineering and medical experts who can give you advice with how to solve complex problems.
 
 **If you have any questions, go to the [Slicer forum](https://discourse.slicer.org) and ask us!**
+
+## Glossary
+
+Terms used in various fields of medical and biomedical image computing and clinical images are not always consistent. This section defines terms that are commonly used in 3D Slicer, especially those that may have different meaning in other contexts.
+
+- Annotation: Simple geometric objects and measurements that the user can place in viewers. Annotations module can be used to create such objects, but it is deprecated - being replaced by markups.
+- Bounds: Describes bounding box of a spatial object along 3 axes. Typically defined by 6 floating-point values: X_min, X_max, Y_min, Y_max, Z_min, Z_max.
+- Brightness/contrast: Specifies linear mapping of voxel values to brightness of a displayed pixel. Brightness is the linear offset, contrast is the multiplier.
+- Cell: Data cells are simple topological elements like lines, polygons, tetrahedra, etc.
+- Coordinate system (or coordinate frame, reference frame, space): Specified by position of origin, axis directions, and distance unit. All coordinate systems in 3D Slicer are right-handed.
```

Comment:
In a former post, it i recall you also described the pre and post multiple convention used in ITK and VTK, and the relationship with computer graphics and image processing (if I recall). I am wondering if we could integrate that info in the glossary also.

Reply:
I've added the resampling and modeling transform to the updated version. There is a lot to tell about coordinate systems and transformations, so it could make sense to add a new "Transformations" section either as a new page or as a section in "Registration" page and refer to that for details.

Reply:
Agreed, we don't want too much in the glossary.  The collaborative writing process is working well and I don't mind adding or reviewing blocks of material.  [This coordinate systems wiki page](https://www.slicer.org/wiki/Coordinate_systems) gets cited a lot, so maybe we incorporate that and add transform material.